ONSdigital/eq-survey-runner
# Eq 496 typography

### What is the context of this PR?

Updated typography to match prototype by adding 'Lato' font and giving all elements class names from the planetary 'Typography matrix' found in the Survey runner style guide.
### How to review
1. Checkout this branch
2. Use the schema `0_basetheme.json`
3. Check 'Lato' is being used throughout the survey and that planet class names appear on these elements: 
   • Headings: `<h1 class=“planetclassname”>` - `<h6 class=“planetclassname”>`
   • Paragraph text: `<p class=“planetclassname”>`
   • Labels: `<label class=“planetclassname”>` and `<span class="label__inner planetclassname">`
   • Buttons: `<button class=“planetclassname”>`
   • Error Alerts: `<li class="list__item planetclassname”>`
   • Summary page
   • Error page
   • Thank you page
### Who worked on the PR

@bennpearson, @hamishtaplin
